src/liboslexec/llvm_util.cpp

```diff
@@ -2500,6 +2500,11 @@ LLVM_Util::type_ptr (llvm::Type *type)
     return llvm::PointerType::get (type, 0);
 }
 
+llvm::Type *
+LLVM_Util::type_wide(llvm::Type *type)
+{
+    return llvm::VectorType::get(type, m_vector_width);
```

Comment:
This fails to build in LLVM 11+, so call existing static LLVM_Util helper that handle the LLVM versioning for us
```
    return llvm_vector_type(type, m_vector_width);
```

Reply:
Done.